plugin-runscanner/src/main/java/ca/on/oicr/gsi/shesmu/runscanner/RunScannerClient.java

```diff
@@ -63,6 +62,8 @@ private static final class RunInformation {
     private final int lanes;
     private final Optional<Long> readEnds;
     private final String serialNumber;
+    private final Optional<PipelineRun> analysis;
```

Comment:
Why `analysis` instead of `pipelineRun`?

Reply:
What's wrong with analysis?

Reply:
"analysis" is an overloaded term: "the output files" (as in Vidarr),  "running workflow(s) to generate files" (as in "running analysis"), and others. "pipelineRun" is more narrowly scoped. It's also the term that runscanner uses, and it's good to keep our terminology consistent, especially when it's for a concept that is somewhat new to our team.

Reply:
Don't think it matters too much since the type is still PipelineRun, and also the results of DRAGEN are still analysis in the senses you mention: output files from a bioinformatics workflow. but since it doesn't matter too much, done